src/Resources/app/storefront/src/paypal-v2/payone-payment.paypal-v2.js

```diff
@@ -28,7 +28,19 @@ export default class PayonePaymentPayPalV2 extends Plugin {
     }
 
     _loadScript() {
-        const queryString = `client-id=${this.options.clientId}&merchant-id=${this.options.merchantId}&currency=${this.options.currency}&intent=authorize&locale=${this.options.locale}&commit=true&vault=false&disable-funding=card,sepa,bancontact${this.options.showPayLaterButton ? '&enable-funding=paylater&commit=false' : ''}`;
+        const queryString = new URLSearchParams({
+            'client-id': this.options.clientId,
+            'merchant-id': this.options.merchantId,
+            'currency': this.options.currency,
+            'intent': 'authorize',
+            'locale': this.options.locale,
+            'commit': 'false',
+            'vault': 'false',
+            'disable-funding': 'card,sepa,bancontact',
+            ...(this.options.showPayLaterButton ? {
+                'enable-funding': 'paylater',
+            } : {}),
+        }).toString();
```

Comment:
did you missed the `commit: false`?

In your present change, it was present.
If not, you can resolve this conversation.